proposals/lip-0045.md

```diff
@@ -467,6 +471,7 @@ channelDataSchema = {
   * `size`: The current size of the tree, i.e. the number of cross-chain messages sent to the partner chain. The default value of this property is 0.
 * `partnerChainOutboxRoot`: The value of this property is set to the outbox root computed from the last CCU from the partner chain containing some cross-chain messages. It is used to validate the cross-chain messages contained in a future CCU when the CCU does not certify a new outbox root. The default value of this property is the constant `EMPTY_HASH`.
 * `messageFeeTokenID`: This property is the token ID of the token used to pay for the cross-chain message fees. The value used in channels between mainchain and sidechains is `messageFeeTokenID = Token.getTokenIDLSK()`, corresponding to the LSK token.
+* `minReturnFeePerByte`: This property is the minimum fee per byte to automatically send back a CCM from the partner chain in case of exeuction errors (see the [`bounce`](#bounce) function). In particular, the CCM fee must be larger or equal than the product of its size in bytes and `minReturnFeePerByte`. The value used in channels between mainchain and sidechains is `minReturnFeePerByte = MIN_RETURN_FEE_PER_BYTE`.
```

Comment:
Question: Would it be the same for channels between sidechains where `messageFeeTokenID = LSK`? (of course in direct channels, we will require this value as part of the registration trs, just wondering if the value would be the same). Then we could say that this value is "mandatory" for all channels where message fees are paid in LSK.

Reply:
in a direct channel the messageFeeTokenID cannot be LSK

Reply:
ah right since it should be native to one of the two chains.Fine, resolving this comment.